documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/SlobrokPolicy.java

```diff
@@ -33,7 +33,7 @@
                         }
                     } catch (InterruptedException e) {
                     }
-                    firstTry = true;
+                    firstTry = false;
```

Comment:
![The best](https://media1.tenor.com/images/c3a76bdc54331bed2c47a7381605463c/tenor.gif)